leakcanary-android-core/src/main/java/leakcanary/AndroidReachabilityInspectors.kt

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress("DEPRECATION")
+
```

Comment:
For the non-Support Library `Fragment`

Reply:
Make it local instead?

Reply:
Either way; saw other `@file` uses so went with that so I wouldn't need to do it for the import as well.